app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -31,3 +32,36 @@ const deserialize = (data) => {
     stories: projectBoard.stories.map(Story.deserialize)
   }
 }
+
+export const hasSearch = projectBoard => Boolean(projectBoard.search.keyWord);
+
+export const validSearch = keyWord => Boolean(keyWord.trim());
+
+export const isOperandSearch = word => {
+  const operandsWords = Object.keys(operands);
+
+  return operandsWords.some(operand => word.includes(operand)) && word.includes(':');
+}
+
+export const serializeKeyWordSearch = keyWord => {
+  if (isOperandSearch(keyWord)) {
+    const arraySearch = keyWord.split(':');
+    return `${operands[arraySearch[0]] || arraySearch[0]}:${arraySearch[1]}`;
+    const [firstKeyWord, secondKeyWord] = keyWord.split(':');
+
+    if (!isEnglishLocale()) {
+      const translations = _.invert(I18n.translations[currentLocale()].story[firstKeyWord]);
+      const translatedWord = I18n.t(`story.${firstKeyWord}.${translations[secondKeyWord]}`, { locale: I18n.defaultLocale });
+      return `${operands[firstKeyWord] || translatedWord}:${translatedWord}`;
+    } else {
+      return `${operands[firstKeyWord] || secondKeyWord}:${secondKeyWord}`;
+    }
+
+  }
+
+  return keyWord;
+}
+
+const isEnglishLocale = () => I18n.currentLocale() === 'en';
```

Comment:
Since you create `currentLocale` function, you could use it here:
```suggestion
const isEnglishLocale = () => currentLocale() === 'en';
```